src/uct/tcp/sockcm/sockcm_md.c

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2017.  ALL RIGHTS RESERVED.
+ * Copyright (C) NVIDIA Corporation. 2019.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#include "sockcm_md.h"
+
+#define UCT_SOCKCM_MD_PREFIX              "sockcm"
+
+static ucs_config_field_t uct_sockcm_md_config_table[] = {
+  {"", "", NULL,
+   ucs_offsetof(uct_sockcm_md_config_t, super), UCS_CONFIG_TYPE_TABLE(uct_md_config_table)},
+  {NULL}
+};
+
+static void uct_sockcm_md_close(uct_md_h md);
+
+static uct_md_ops_t uct_sockcm_md_ops = {
+    .close                  = uct_sockcm_md_close,
+    .query                  = uct_sockcm_md_query,
+    .is_sockaddr_accessible = uct_sockcm_is_sockaddr_accessible,
```

Comment:
can use `ucs_empty_function_return_zero` casted to the correct type. or it will be used in the next PR?

Reply:
yes, this seems as a skeleton for next PR